pkg/datapath/iptables/iptables.go

```diff
@@ -1226,6 +1257,14 @@ func (m *IptablesManager) InstallRules(ifName string) error {
 		}
 	}
 
+	if skipPodToPodConntrack(false) {
+		podsCIDR := option.Config.IPv4NativeRoutingCIDR().String()
+
+		if err := m.addNoTrackPodToPodRules("iptables", podsCIDR); err != nil {
+			return fmt.Errorf("Cannot install rules to skip pod to pod CT: %s", err)
```

Comment:
For error messages, use `%w`. The detailed motivation was in [this post](https://blog.golang.org/go1.13-errors). Sorry I missed this on the previous review :/.
```suggestion
			return fmt.Errorf("Cannot install rules to skip pod to pod CT: %w", err)
```